lib/widget/carousel.dart

```diff
@@ -153,11 +159,11 @@ class CarouselState extends WidgetState<Carousel> with TemplatedWidgetState {
     );
 
     // show indicators
-    if (widget._controller.indicatorType != null && widget._controller.indicatorType != IndicatorType.none) {
+    if (widget._controller.indicatorType != null && widget._controller.indicatorType != IndicatorType.none || customIndicator != null || selectedCustomIndicator != null) {
```

Comment:
don't mix && and || without parenthesis. Be explicit about your intention, as different languages evaluate them differently.

Reply:
I updated it like this

```dart
bool isCustomIndicator =
        customIndicator != null || selectedCustomIndicator != null;
    if (widget._controller.indicatorType != null &&
            widget._controller.indicatorType != IndicatorType.none ||
        isCustomIndicator) {
```